pex/resolver.py

```diff
@@ -329,6 +342,10 @@ def resolve(requirements,
     ``context``.
   :keyword allow_prereleases: (optional) Include pre-release and development versions.  If
     unspecified only stable versions will be resolved, unless explicitly included.
+  :keyword pkg_blacklist: (optional) A blacklist dict (str->str) that maps package name to
```

Comment:
would love to have a better documentation on why a package needed to be blacklisted